design/open/uncomitted-gc.md

```diff
@@ -153,8 +150,16 @@ Optimized run uses the previous GC run output, to perform a partial scan of the
    2. Subtract current `Uncommitted DF` from `Store DF`
    3. Filter files in special paths
    4. The remainder is a list of files which can be safely removed
-3. Save run information under the GC path  
-   See previous for steps
+3. Save run data (see: [GC Saved Information](#gc-saved-information))
+
+### GC Saved Information
+
+For each GC run, save the following information using the GC run id as detailed in this [proposal](https://github.com/treeverse/cloud-controlplane/blob/main/design/accepted/gc-with-run-id.md):
+1. Save metaranges in `_lakefs/gc/run_id/metadata/`
+2. Save `Uncommitted DF` in `_lakefs/gc/run_id/uncommitted.parquet`
```

Comment:
Correct me if I'm wrong @treeverse/ecosystem, but parquet outputs are multiple files (per column) and it's normally saved under a prefix of it's own for better separation, right?
So a better path would be `_lakefs/gc/run_id/uncommitted/entries.parquet`